tests/adapter/dbt/tests/adapter/utils/test_timestamps.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from dbt.tests.util import check_relation_has_expected_schema, run_dbt
+
+_MODEL_CURRENT_TIMESTAMP = """
+SELECT {{current_timestamp()}} as current_timestamp,
+       {{current_timestamp_in_utc()}} as current_timestamp_in_utc
+"""
```

Comment:
We typically [lowercase `select`](https://github.com/dbt-labs/corp/blob/4b553082bf0fb8e285934195331925461bef4784/dbt_style_guide.md?plain=1#L100) and use [spaces inside Jinja delimiters](https://github.com/dbt-labs/corp/blob/4b553082bf0fb8e285934195331925461bef4784/dbt_style_guide.md?plain=1#L236).
```suggestion
_MODEL_CURRENT_TIMESTAMP = """
SELECT {{ current_timestamp() }} as current_timestamp,
       {{ current_timestamp_in_utc() }} as current_timestamp_in_utc
"""
```